src/commands/base-command.js

```diff
@@ -10,6 +11,7 @@ const merge = require('lodash/merge')
 const jsClient = import('netlify')
 const netlifyConfigPromise = import('@netlify/config')
 
+const { watchDebounced } = require('../lib/functions/watcher')
```

Comment:
Should we move this `watcher` utility to a more generic location?
It can be confusing to have our base command use a functions utility.
We have a `command-helpers.js` file that can be more suitable.

Reply:
done in bb026a5b9d17555aa1f1d9170abf4f5d12c68398